### DISCUSSION TITLE
Wrong data product link

### DISCUSSION THREAD

**Discussion by pramodkvarma** [Sunbird-Obsrv/Community#5] [Issues]
@rhwarrier https://obsrv.sunbird.org/learn/product-and-developer-guide/data-service page has a PDF link to DIKSHA data exhaust doc! Please fix.

**Comment by rhwarrier** [comment]
Sure, @pramodkvarma  - this was a stop gap arrangement. Will redo the doc for SB and have it changed on the microsite

**Reply by pramodkvarma** [reply]
Thanks @rhwarrier. Please create/convert this into an issue tracker (GitHub issue) item and track.

**Comment by rhwarrier** [comment]
The document has been updated.
